modules/pel/peltool/peltool.py

```diff
@@ -713,68 +713,96 @@ def main():
     PELsPath = "/var/lib/phosphor-logging/extensions/pels/logs/"
     PELsArchivePath = "/var/lib/phosphor-logging/extensions/pels/logs/archive"
     inBMC = os.path.isdir(PELsPath)
-    parser = argparse.ArgumentParser(formatter_class=CustomFormatter, description="PELTools")
+    example_text = '''Examples:
+        -------------------------
+        List PELs based on the severity: peltool -l -S Predictive
```

Comment:
```
peltool -l -S Predictive
```
@harsh-agarwal1  correct me if I'm wrong, but doesn't this one add in the severity to the search?  and I think since Predictive is in there by default, it'd be better to add in something like Informational to show how it includes new things.